data.sql

```diff
@@ -38,3 +38,11 @@ WHERE Name = 'Charmander' or Name = 'Squirtle' or Name = 'Blossom';
 UPDATE animals
 SET owner_id = 5
 WHERE Name = 'Angemon' or Name = 'Boarmon';
+
+INSERT INTO vets (name,age,date_of_graduation) VALUES('Willian Tatcher', 45, (to_date('04/23/2000','mm/dd/yyyy')) ),('Maisy Smith', 26, (to_date('01/17/2019','mm/dd/yyyy')) ),('Stephanie Mendez', 64, (to_date('05/04/1981','mm/dd/yyyy')) ),('Jack Harkness', 38, (to_date('06/08/2008','mm/dd/yyyy')) );
+
+INSERT INTO specialization VALUES(1, 1 ),(1, 3 ), (2, 3 ),(2, 4 );
```

Comment:
- [ ] Let us modify this implementation by selecting id from vets and id from species. Your implementation is very complicated to know what exactly are you trying to insert into database.

Hint:  `INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));`

Reply:
Thanks for the alternative. Done